--- DxLibMake/Windows/DxBaseImageWin.cpp	2023-02-26 00:06:48.000000000 +0900
+++ DxLibMake/Windows/DxBaseImageWin_patch.cpp	2023-06-25 10:59:02.749551000 +0900
@@ -167,7 +167,7 @@
 	else
 	if( _MEMCMP( &WICPixelFormatGUID, &GUID_WICPIXELFORMAT32bppRGB, sizeof( GUID ) ) == 0 )
 	{
-		NS_CreateColorData( &BaseImage->ColorData, 32, 0x0000ff, 0x00ff00, 0xff0000, 0 ) ;
+		NS_CreateColorData( &BaseImage->ColorData, 32, 0x0000ff, 0x00ff00, 0xff0000, 0, 0, 0, FALSE ) ;
 	}
 	else
 	if( _MEMCMP( &WICPixelFormatGUID, &GUID_WICPIXELFORMAT24bppBGR, sizeof( GUID ) ) == 0 )
